Makefile

```diff
@@ -1,35 +1,40 @@
-CFLAGS+=$(shell pkg-config --cflags cairo-xlib xinerama glib-2.0 xext x11 xtst 2> /dev/null || echo -I/usr/X11R6/include -I/usr/local/include)
-LDFLAGS+=$(shell pkg-config --libs cairo-xlib xinerama glib-2.0 xext x11 xtst 2> /dev/null || echo -L/usr/X11R6/lib -L/usr/local/lib -lX11 -lXtst -lXinerama -lXext -lglib)
-LDFLAGS+=$(shell pkg-config --libs glib-2.0)
+CFLAGS+=$(shell pkg-config --cflags cairo-xlib 2> /dev/null)
+CFLAGS+=$(shell pkg-config --cflags xinerama 2> /dev/null)
+CFLAGS+=$(shell pkg-config --cflags glib-2.0 2> /dev/null)
+CFLAGS+=$(shell pkg-config --cflags x11 2> /dev/null)
 
-OTHERFILES=README CHANGELIST COPYRIGHT \
+LDFLAGS+=$(shell pkg-config --libs cairo-xlib 2> /dev/null)
+LDFLAGS+=$(shell pkg-config --libs xinerama 2> /dev/null)
+LDFLAGS+=$(shell pkg-config --libs glib-2.0 2> /dev/null)
+LDFLAGS+=$(shell pkg-config --libs x11 2> /dev/null)
+
+PREFIX=/usr
+
+OTHERFILES=README.md CHANGELIST COPYRIGHT keynav.pod \
            keynavrc Makefile version.sh VERSION
-#CFLAGS+=-DPROFILE_THINGS
-#LDFLAGS+=-lrt
 
 VERSION=$(shell sh version.sh)
 
 #CFLAGS+=-pg -g
 #LDFLAGS+=-pg -g
-#LDFLAGS+=-L/usr/lib/debug/usr/lib/ -lcairo -lX11 -lXinerama -LXtst -lXext
 #CFLAGS+=-O2
 
 #CFLAGS+=-DPROFILE_THINGS
 #LDFLAGS+=-lrt
 
-.PHONY: all
+.PHONY: all uninstall
 
 all: keynav
 
 clean:
-	rm *.o keynav keynav_version.h || true;
+	rm -f *.o keynav keynav_version.h keynav.1.gz
 
 keynav.o: keynav_version.h
 keynav_version.h: version.sh
 
 keynav: LDFLAGS+=-Xlinker -rpath=/usr/local/lib
 keynav: keynav.o
-	$(CC) keynav.o -o $@ $(LDFLAGS) -lxdo; \
+	$(CC) keynav.o -o keynav $(CFLAGS) $(LDFLAGS) -lxdo
```

Comment:
why change $@ to `keynav` ? `$@` is the target name, which is `keynav` in this case. I'm a little confused; can you help me understand?

Reply:
Also not my change, pinging @yjftsjthsd-g

Reply:
My memory fails me, unfortunately. I may have changed it just because `$@` was unfamiliar to me; looking now, it appears to be reasonably portable (the current version of the Makefile only works in GNU make, and I'd like to fix that), so I agree that it should be reverted. The only other minor concern would be use in ex. the debug target, but building a differently named debug binary isn't exactly a bad thing.